src/java.desktop/share/classes/javax/swing/text/html/CSS.java

```diff
@@ -2968,7 +2968,7 @@ ImageIcon getImage(URL base) {
 
         @Override
         public int hashCode() {
-            return (this.svalue != null) ? Objects.hashCode(this.svalue) : 0;
+            return Objects.hashCode(this.svalue);
```

Comment:
```suggestion
            return Objects.hashCode(svalue);
```
Using `this` is also redundant.